app/application/dto/account_dto.py

```
@@ -2,15 +1,14 @@
+from pydantic import BaseModel
 
 
-@dataclass
-class BalanceDTO:
+class BalanceDTO(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)